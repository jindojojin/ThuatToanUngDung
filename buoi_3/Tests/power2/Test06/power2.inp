10
4096 12 145 23456 6789654 123456 1048576 156 234567 1
