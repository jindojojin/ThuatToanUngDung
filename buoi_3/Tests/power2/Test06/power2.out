1 4096 1048576
